### ISSUE TITLE
Mounting partitions fails

### ISSUE THREAD

**Issue by splatmat** [meefik/linuxdeploy#61]
I've got an Odys Neo X7 running a rooted Android 4.0.3 Kernel Version 3.0.8+ and i tried to install Kali with Linux Deploy. I use an 8GB FAT32 sd card to install the linux.img file. Installation fails after making file system. 

i get the following output:

Mounting partitions: 
/ ... fail
<<< end: install

What am i doing wrong? or is it a known issue? i really need help here...

**Comment by kubajunki** [comment]
So I am a super noob but i managed to solve the problem. All i had to do is to change File System to ext2 ( or whatever works), image size = 900, installed busybox, and uncheck built- in shell. also if you want you can change the directory of the file.

**Comment by travisjayday** [comment]
I had to go to busybox and check all the commands at the bottom (all green) and install them. It seems as if busybox left out some important commands such as 'dd' 'mount' and 'unmount'.

**Comment by VADemon** [comment]
I had the same issue, with the help of the debug option I found out that the su binary was missing from bin/ but it was present in /xbin and besides that xbin/ was set to be used instead of bin/
TL;DR: make sure your BusyBox installation is 100% correct and you're not missing any binaries from either folder.

**Comment by lpreams** [comment]
I was having this issue after trying to use Directory installation instead of File installation. I think at some point a loop device got stuck mounted and it wasn't able to connect a new image to the loop device. I rebooted the phone and it was all good from there. I assume the loop devices got cleared out and that's what fixed it for me.

**Comment by iacchus** [comment]
For me this issue was happening because of SuperSU 'Mount namespace separation'. Go to settings and disable and reboot.

**Comment by Roevin** [comment]
u0_a194@android:/ $
u0_a194@android:/ $ cd /storage/emulated/0/Kalilinux
u0_a194@android:/storage/emulated/0/Kalilinux $ su
s.linuxonandroid/files/bootscript.sh /storage/emulated/0/Kalilinux/kalilinux.img                  <
Checking loop device... MISSING
Creating loop device... OK
mount: mounting /storage on /data/local/mnt/external_sd failed: Invalid argument
No user defined mount points
net.ipv4.ip_forward = 1
chroot: can't execute '/root/init.sh': Permission denied
Shutting down Linux ARM
umount: can't umount /data/local/mnt/external_sd: Invalid argument
umount: can't umount /data/local/mnt/dev: Invalid argument
1|root@android:/storage/emulated/0/Kalilinux #
I got this problem while installing kali linux on my tab 3. Any help

**Comment by ThatGuyMelo** [comment]
Sorry to be touching on an old topic but i've been struggling with this issue, i'd really appreciate help tackling it.

[03:28:22] Updating configuration file ... done
[03:28:22] >>> configure
[03:28:22] Mounting partitions: 
[03:28:22] / ... fail
[03:28:22] <<< configure
[03:28:25] Updating configuration file ... done
[03:28:25] >>> start
[03:28:25] Mounting partitions: 
[03:28:25] / ... fail
[03:28:25] <<< start

**Comment by ifebonboi** [comment]
Same problem. After creatin d image. Bt i get mounting partition fail.  I really need help here

**Comment by WilsonBradley** [comment]
Has there been a successful story on doing this? I currently have a 4gb loop file almost full. Would like to use GParted to split up the SD with lets say a 15gb ext4 partion and copy the loop to that partition and expand it. 
I still don't understand how to make Linux Deploy see that new partition on SD.

Any thoughts on this?

Thanks
